src/docs/reference/modules/Scripting/README.md

```diff
@@ -8,42 +8,48 @@ The scripting module provides an API allowing you to evaluate custom scripts in
 
 ### Executing some script
 
-The main interface is `IScriptingManager`.
+The main interface is [IScriptingManager](https://github.com/OrchardCMS/OrchardCore/blob/main/src/OrchardCore/OrchardCore.Infrastructure.Abstractions/Scripting/IScriptingManager.cs).
 
 ```csharp
 public interface IScriptingManager
```

Comment:
Why we need the class here while you reference it above?

Reply:
I think it's helpful to see the docs that are in the file. Originally I left the reference only so if the code/docs changes, we don't have to maintain the code in multiple places. I don't know what is acceptable to you, but the PR is to make it where I though would be useful.

Reply:
The reference for `IScriptManager` is good, my comment was about `IScriptingEngine` no need to reference the code.

Reply:
To make sure I understand the ask, you want me to:
1) Remove the GitHub reference to `IScriptingEngine`
2) Keep the GitHub reference to `IScriptingManager`? 
3) Keep the following code as part of this doc?
```csharp
{
    IScriptingEngine GetScriptingEngine(string prefix);
    object Evaluate(string directive, IFileProvider fileProvider, string basePath, IEnumerable<IGlobalMethodProvider> scopedMethodProviders);
    IReadOnlyList<IGlobalMethodProvider> GlobalMethodProviders { get; }
}
```

Reply:
Exactly except you need to remove `IScriptManager` code because you referenc to it